fog/types/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) 2018-2022 The MobileCoin Foundation
 
-#![no_std]
+#![cfg_attr(not(test), no_std)]
```

Comment:
not sure if this `not(test)` snuck in here or if this was one of those things going to be fixed in a later PR
```suggestion
#![no_std]

Reply:
Done